code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/EdgePersonalizationResponseHandler.java

```diff
@@ -641,10 +734,11 @@ private void updatePropositionInfo(
     }
 
     @SuppressWarnings("NestedForDepth")
-    private Map<Surface, List<Proposition>> getPropositionsFromFeedRulesEngine(final Event event) {
+    private Map<Surface, List<Proposition>> getPropositionsFromContentCardRulesEngine(
```

Comment:
This method is also called inside the `serialWorkDispatcher`. We should ensure thread safety for map and list reads and writes